services/storage/storage.go

```diff
@@ -15,7 +21,9 @@ type VirtualHardDiskProperties struct {
 	Dynamic *bool `json:"dynamic,omitempty"`
 	// Blocksizebytes - TODO: Revisit exposing this
 	Blocksizebytes *int32 `json:"blocksizebytes,omitempty"`
-	//Logicalsectorbytes - TODO: Revisit exposing this
+	// HyperVGeneration - Gets the HyperVGenerationType of the VirtualMachine created from the image. Possible values include: 'HyperVGenerationTypesV1', 'HyperVGenerationTypesV2'
```

Comment:
Looks like code alignment is messed up. Check if you editor tab is mapped to 4 spaces.